ckanext/push_errors/error_logger.py

```diff
@@ -0,0 +1,27 @@
+import traceback
+import logging
+
+
+log = logging.getLogger(__name__)
+
+logged_errors = set()  # To track already logged errors
+disabled_notifications = set()  # To track disabled error notifications
+
+
+def log_error(error, disable_notification=False):
```

Comment:
Drop this this file and this function from other places and tests
This PR will be only to generate a unique ID, not to monitor/disable calls